reps/2025-03-18-label-based-scheduling.md

```diff
@@ -0,0 +1,212 @@
+## Summary
+### General Motivation
+
+This REP summarizes the current state of the node label scheduling feature enhancement and the remaining work to fully support label based scheduling for Tasks and Actors in Ray.
```

Comment:
```suggestion
This REP summarizes the current state of the node label scheduling feature enhancement and the remaining work to fully support scheduling using label selectors in Ray.
```

Let's center on the term "label selector" rather than "label-based scheduling."